corehq/apps/fixtures/tests.py

```diff
@@ -338,6 +339,14 @@ def test_combo(self):
         bad_name = "<space>\<dadgg sd"
         self.assertTrue(is_identifier_invalid(bad_name))
 
+    def test_starts_with_number(self):
+        bad_name = "0hello"
+        self.assertTrue(is_identifier_invalid(bad_name))
+
+    def test_unicode(self):
+        bad_name = u"ﾉｲ丂 ﾑ ｲ尺ﾑｱ! \_(ツ)_/¯"
+        self.assertTrue(is_identifier_invalid(bad_name))
```

Comment:
Completely agree. I actually first wrote it to return `False` when it was bad, but then all the tests failed...